contracts/credit/CreditFacadeV3.sol

```diff
@@ -324,26 +317,26 @@ contract CreditFacadeV3 is ICreditFacadeV3, ACLNonReentrantTrait {
         FullCheckParams memory fullCheckParams = _multicall(
             creditAccount, calls, collateralDebtData.enabledTokensMask, LIQUIDATE_CREDIT_ACCOUNT_FLAGS, skipCalls
         ); // U:[FA-16]
-        collateralDebtData.enabledTokensMask &= fullCheckParams.enabledTokensMaskAfter;
+        collateralDebtData.enabledTokensMask &= fullCheckParams.enabledTokensMaskAfter; // U:[FA-16]
 
         bool success = BalancesLogic.compareBalances({
             creditAccount: creditAccount,
             tokensMask: collateralDebtData.enabledTokensMask,
             balances: initialBalances,
             comparison: Comparison.LESS
         });
-        if (!success) revert RemainingTokenBalanceIncreasedException(); // U:[FA-16]
+        if (!success) revert RemainingTokenBalanceIncreasedException(); // U:[FA-14]
 
-        collateralDebtData.enabledTokensMask = collateralDebtData.enabledTokensMask.enable(UNDERLYING_TOKEN_MASK);
+        collateralDebtData.enabledTokensMask = collateralDebtData.enabledTokensMask.enable(UNDERLYING_TOKEN_MASK); // U:[FA-16]
 
         (uint256 remainingFunds, uint256 reportedLoss) = ICreditManagerV3(creditManager).liquidateCreditAccount({
             creditAccount: creditAccount,
             collateralDebtData: collateralDebtData,
             to: to,
-            isExpired: isExpired
-        }); // U:[FA-16]
+            isExpired: !isUnhealthy
```

Comment:
At this point, since it didn't revert earlier, the account is either unhealthy, expired, or both. Expired liquidation is only performed when it's not unhealthy (this is equivalent to what we had earlier, just looks neater).